randovania/data/json_data/prime3.txt

```diff
@@ -630,6 +647,9 @@ Asset id: 8948533108801763587
 > Dock to Generator Shaft; Heals? False
   * Normal Door to Generator Shaft/Dock to Generator C
 
+> Event - Meta Ridley; Heals? False
+  * Event Meta Ridley
+
```

Comment:
For Generator C (above): Missing connections. Minor, and might not be relevant.

Reply:
For now, I moved the meta ridley event to Generator Shaft. I made Generator C Access -> Generator Shaft require morph ball bombs.